packages/video_player/ios/Classes/VideoPlayerPlugin.m

```diff
@@ -352,6 +352,19 @@ - (void)setVolume:(double)volume {
   _player.volume = (float)((volume < 0.0) ? 0.0 : ((volume > 1.0) ? 1.0 : volume));
 }
 
+- (void)setSpeed:(double)speed {
+  if (speed == 1.0 || speed == 0.0) {
+    _player.rate = speed;
+  } else if (speed < 0 || speed > 2.0) {
+    NSLog(@"Speed outside supported range %f", speed);
```

Comment:
Could you throw a `FlutterError` here like we do in other plugins.
https://github.com/flutter/plugins/blob/master/packages/camera/ios/Classes/CameraPlugin.m#L82
https://github.com/flutter/plugins/blob/master/packages/camera/ios/Classes/CameraPlugin.m#L7